src/main/java/com/google/jenkins/plugins/computeengine/ComputeEngineCloud.java

```diff
@@ -187,14 +207,46 @@ public String getProjectId() {
     return projectId;
   }
 
-  public void addConfiguration(InstanceConfiguration config) {
-    configurations.add(config);
+  /**
+   * Set configurations for this cloud.
+   *
+   * @param configurations configurations to be used
+   */
+  @DataBoundSetter
+  public void setConfigurations(List<InstanceConfiguration> configurations) {
+    if (this.configurations == null) {
```

Comment:
This if block seems redundant as configurations is having its value set directly after.

Reply:
yup.. I forgot `else` :)
Maybe its not needed..

Reply:
Removed..